cloudflare.php

```diff
@@ -468,6 +469,16 @@ function purgeCache()
     }
 }
 
+function sslRewrite() {
+    if ( isset( $_SERVER['HTTP_X_FORWARDED_PROTO'] ) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https' ) {
+        $_SERVER['HTTPS'] = 'on';
+
+        return true;
+    }
+
+    return false;
```

Comment:
We can not write tests for sslRewrite if it's in `cloudflare.php`. It'll need to be moved inside `src/WordPress`.